### ISSUE TITLE
Intellij plugin docs seem incorrect

### ISSUE THREAD

**Issue by buggtb** [gitpod-io/gitpod#9902]
### Bug description

According to the docs you can add

```
jetbrains:
  intellij:
    prebuilds:
      version: stable

```
and 

```
jetbrains:
  intellij:
    - zielu.gittoolbox
    - izhangzhihao.rainbow.brackets
```

to your gitpod yaml file, but surely the plugins block is missing a plugins descriptor or something? Cause that doesn't look like it would make a valid yaml file.

### Steps to reproduce

Read the docs

**Comment by loujaybee** [comment]
If you need @buggtb, the JSON schema is here: https://gitpod.io/schemas/gitpod-schema.json

We are also missing either: 
1. Updating the main reference page -> https://www.gitpod.io/docs/references/gitpod-yml 
2. Note about current state of the property: https://www.gitpod.io/docs/ides-and-editors/intellij#indexing-using-prebuilds

**Comment by andreafalzetti** [comment]
Thanks for flagging. Earlier today I've [fixed](https://github.com/gitpod-io/website/issues/2014) the docs adding the `plugins` key, the change is already live.

**Comment by akosyakov** [comment]
https://github.com/gitpod-io/website/issues/2003 is a dup?

**Comment by andreafalzetti** [comment]
Closing as the original issue has been fixed - we'll follow up with #2003 cc @akosyakov @loujaybee
